public/static/locales/en/common.json

```diff
@@ -98,8 +98,10 @@
     "logout": "Sign out"
   },
   "alert": {
-    "dismissAlert": "Close this alert"
+    "dismissAlert": "Close this alert",
+    "dismissNotification": "Close this alert"
```

Comment:
Is it normal that both `dismissAlert` and `dismissNotification` have the same value? If yes then we could just use one of them.

Reply:
Just going to use dismissAlert --- it's for an aria label should be fine